samples/install-without-bom/pom.xml

```diff
@@ -42,7 +42,12 @@
     <dependency>
       <groupId>org.apache.arrow</groupId>
       <artifactId>arrow-vector</artifactId>
-      <version>0.17.1</version>
+      <version>1.0.1</version>
+    </dependency>
+    <dependency>
+      <groupId>org.apache.arrow</groupId>
+      <artifactId>arrow-memory-netty</artifactId>
+      <version>1.0.1</version>
```

Comment:
Do you want to use a property for this? Otherwise, you will get separate renovate PRs for each of these deps

Reply:
sure - done